org.gnome.Eolie.json

```diff
@@ -0,0 +1,314 @@
+{
+  "app-id": "org.gnome.Eolie",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "eolie",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=host",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:rw",
+    "--talk-name=org.freedesktop.secrets",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+  ],
+  "modules": [
+    {
+      "name": "gst-libav",
```

Comment:
Well my second point was GStreamer already has various codecs packaged so we don't need to rebuild them like vp9, opus, ogg, should work ootb. (we need to add mp3 to that list too)

As for the theory that websites can use any codec.. sure but the reality is a dozen are.